pkg/model/awsmodel/autoscalinggroup.go

```diff
@@ -211,6 +212,11 @@ func (b *AutoscalingGroupModelBuilder) buildLaunchTemplateTask(c *fi.ModelBuilde
 		case kops.SubnetTypePrivate:
 			lt.AssociatePublicIP = fi.Bool(false)
 		}
+
+		// @step: add an IPv6 address
+		if subnets[0].IPv6CIDR != "" {
```

Comment:
Shouldn't we do this if any of the subnets have an IPv6 CIDR? `GatherSubnets` enforces that there can't be multiple subnet types, but I think it is possible for some but not all subnets to have an IPv6 address.

Reply:
Good point, done.